MavenQuickStart/quickApp/src/main/java/com/globant/topicthree/exerciseone/ShoppingCart.java

```diff
@@ -0,0 +1,56 @@
+package com.globant.topicthree.exerciseone;
+
+import java.io.Serializable;
+import java.util.LinkedList;
+import java.util.List;
+import java.util.UUID;
+
+/**
+ * This entity represent a virtual Shopping Cart. of this.
+ * 
+ * @author andres.vaninetti
+ *
+ */
+public class ShoppingCart implements Serializable {
+
+	private static final long serialVersionUID = 1L;
+
+	private List<Product> products;
+
+	private String id;
+
+	/**
+	 * Default constructor.
+	 * 
+	 */
+	public ShoppingCart() {
+		this.products = new LinkedList<Product>();
```

Comment:
Ok thanks, in this case I chose LinkedList because is double vinculated and I think therefore is efficient for add and remove elements, but I'll read this doc in order to be more specific to chose. Thanks Marco